history-of-the-quakers-v4/original/11-book-7-chapter-5.adoc

```diff
@@ -124,6 +128,10 @@ N+++.+++ B. These accounts appear to be drawn up about the year 1728.
 In the year 1736,
 they also presented a petition and the annexed case to the parliament then sitting.
 
+[.embedded-content-document.address]
+--
+
+[.letter-heading]
```

Comment:
I think let's just make this a plain `=== ` subheading, since it doesn't seem so much like a letter.